tasks/jsinspect.js

```diff
@@ -24,9 +25,21 @@ module.exports = function(grunt) {
       identifiers: false,
       failOnMatch: true,
       suppress:    100,
-      reporter:    'default'
+      reporter:    'default',
+      configFile:  '.jsinspectrc'
     });
 
+    if (fs.existsSync(options.configFile) && fs.lstatSync(options.configFile).isFile()) {
+      var contents = strip(fs.readFileSync(options.configFile, 'utf8'));
+      var rc = JSON.parse(contents);
```

Comment:
> jsinspect had it but I took away, since thought that if is breaks, it should break..

True - but this is a specific case.
A message like "the json of your config file is not valid please check again".

Is much nicer then

``` 
SyntaxError: Unexpected end of input
SyntaxError: Unexpected string
...
```

I agree that it should still break - but with a nicer Error message e.g. `throw new Error( 'Your config json is not valid...' )`.

What do you think?